src/applications/pact-act/containers/questions/burn-pit/BurnPit-2-1-1.jsx

```diff
@@ -13,12 +13,15 @@ import { pageSetup } from '../../../utilities/page-setup';
 
 export const locationList = (
   <ul>
-    <li>Arabian Sea</li>
-    <li>Gulf of Aden</li>
-    <li>Gulf of Oman</li>
-    <li>The neutral zone on the border between Iraq and Saudi Arabia</li>
-    <li>Persian Gulf</li>
-    <li>Red Sea</li>
+    <li>Bahrain</li>
```

Comment:
The old flow was:

```
Burn Pit 2.1 -> Burn Pit 2.1.1 -> Burn Pit 2.1.2
```

With the new question coming first, it scoots the original 3 questions further down the flow, and now we have:
```
Burn Pit 2.1 -> Burn Pit 2.1.1 -> Burn Pit 2.1.2 -> Burn Pit 2.1.3
```